src/components/autocomplete/QAutocomplete.js

```diff
@@ -148,6 +149,10 @@ export default {
       this.searchId = ''
     },
     setValue (result) {
+      this.enterKey = true
+      if (this.inputEl && this.__input && !this.__input.hasFocus()) {
+        this.inputEl.focus()
```

Comment:
Hmm... Not sure if it useful. When you working with mouse or tuch it does not matter...
It only matters when you work with the keyboard... And there everything is ok...
Besides, double assignment **this.enterKey = true** when Enter press

Reply:
Fixed duplicate this.enterKey = true.
It matters to have it set because it blocks reopening the popover with the list when the value is set in parent.

Reply:
>when the value is set in parent.

Can you explain more detail? I cannot reproduce it... Just wondering is #1282 not fixes this problem?

Reply:
The idea is like this:
- parent component of QAutocomplete has a shadow property that keeps the value as `val` - this is updated according to debouncing and some delays for processing
- if we listen for input from the parent we're going to get in `val` the value before the last key press 
- if we listen for changes in `val` we get the correct value every time, but that value we are sometimes updating from the autocomplete, so we want to skip showing the options in that case

Reply:
Well, its clear ) But we can use target.value from input event? Then there will be a lot of redundant code and checks? I'll play around with this)
I still think it is improperly to use a watcher for autocomplete...